src/components/functional/footer.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  Container, Nav, NavItem, NavLink,
+} from 'reactstrap';
+
+const Footer = () => (
```

Comment:
The footer is still not responsive. 
![Screen Shot 2019-06-12 at 09 36 58](https://user-images.githubusercontent.com/48981919/59328754-df230080-8cf5-11e9-81b7-2427bd93cb4c.png)